torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3111,7 +3121,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
     OpInfo('add',
            dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16),
            assert_autodiffed=True,
-           sample_inputs_func=sample_inputs_add,
+           sample_inputs_func=partial(sample_inputs_binary_pwise, use_alpha=True, alpha=2),
```

Comment:
if the approach suggested above is used, the presence of 'alpha' in the kwargs dict alone indicates that use_alpha=True, so `use_alpha` would be redundant and can be removed

Reply:
Thanks, I didn't know I could do `if x in kwargs` 😞 
But I'm here for learning 😃

Reply:
Yep! I was also confused by that when I was learning Python, because it seems like a magical object. But `kwargs` from `**kwargs` is just a dict.